cost/turbonomics/scale_virtual_machines_recommendations/aws/README.md

```diff
@@ -0,0 +1,21 @@
+# Turbonomics Scale Virtual Machines Recommendations AWS
+
+## What it does
+
+The Turbonomics Scale Virtual Machines Recommendations AWS policy utilizes Turbonomics [Credentials](https://turbonomic.com/api/v3/markets/{market_uuid}/actions) endpoint to provide Scale Virtual Machines Recommendations. From these recommendations we provide monthly savings estimates based on Turbonomics per hour costs.
+
+### Input Parameters
+
+- *Provider* - Cloud provider. Allows AWS, Azure subscriptions or GCP projects.
```

Comment:
This policy only supports AWS, no Azure or GCP.

Reply:
You are right. Thanks for the corrections!